### ISSUE TITLE
Find command bug 

### ISSUE THREAD

**Issue by wjunjie01** [wjunjie01/pe#5]
There seems to have a discrepancy across the UG and the application. 

In the UG, it states that the recipe name (CRITERIA) has to be an exact match.

![image.png](https://raw.githubusercontent.com/wjunjie01/pe/main/files/7ec24ea5-0f7a-400b-a68c-afa1452f1d78.png)

When I do list. The recipe name is 'pizza'.

However, when i do `find kw PIZZA` or `find kw piz` or `find kw              pizza` it seems to return the correct recipe.
Even though they do not match exactly.

TL:DR
PIZZA != pizza
piz != pizza
          pizza != pizza

![image.png](https://raw.githubusercontent.com/wjunjie01/pe/main/files/bf5310e4-0801-4c03-b75e-c4f232d078f7.png)

![image.png](https://raw.githubusercontent.com/wjunjie01/pe/main/files/f771cb17-b8de-468c-ae57-563f9beaf9ae.png)

**Comment by nus-pe-script** [comment]
# Team's Response
Sure, the UG could have been clearer in its guidance for the find command.
# Items for the Tester to Verify
## :question: Issue type

Team chose [`type.DocumentationBug`]
Originally [`type.FeatureFlaw`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
